### ISSUE TITLE
find out about "history" / backups

### ISSUE THREAD

**Issue by nycnoise** [gregsadetsky/nycnoise#181]
a few times now i've accidentally deleted stuff, cuz i'm clumsy & quick heh! minor so far cuz it was single events (when i accidentally hit "save" on an event instead of "save as new" & replace instead of adding; i've thought about it, & the current way of doing things is SO EASY that i don't think i wanna change it even given that possibility!!), BUT i'm a teensy bit concerned about bigger things now that i can't undo revisions...

is there any way to set up periodic backups of the site? or maybe i'll chill if i just learn out about current storage etc. situation?

**Comment by gregsadetsky** [comment]
Great q! We do have automated daily backups for the whole database for the past 7 days

It would def make sense to have a table/system that tracks all changes and allows you to restore i.e. go back in time for any event/venue entry i.e. have an "undo" :)

Let's leave this issue open to come back to this / discuss

**Comment by gregsadetsky** [comment]
This is related to / the same as #223

**Comment by nycnoise** [comment]
(i'm gonna start posting stuff in here again as i think of it, but obvs obvs obvs no pressure / rush!!!)

is there a way for me to access history / a previous version? i did the accidental-"save" thing again whoops heh –– regardless, this is also a reminder for me to add back whatever may 2024 endless life thing for ~The Archive...

<img width="739" alt="Screen Shot 2024-07-01 at 4 14 04 PM" src="https://github.com/gregsadetsky/nycnoise/assets/135757981/2d8d29c5-c507-4e64-9bfa-4f915ccbdb0e">

**Comment by gregsadetsky** [comment]
yeah unfortunately there's no "reverse"/undo right now after save is clicked

a django plugin such as https://django-reversion.readthedocs.io/en/latest/ could be used, but it would have to be tested (just to make sure that it doesn't conflict with our database in some weird/unpredictable way)
